shard.yml

```
@@ -79,10 +82,24 @@ dependencies:
   secrets-env:
     github: spider-gazelle/secrets-env
     version: ~> 1
+  
+  sodium:
+    github: didactic-drunk/sodium.cr
+  
+  tasker:
+    github: spider-gazelle/tasker
+    version: ~> 2.0.3
```

Too tight of a version restriction IMO

```suggestion
    version: ~> 2.0
```